### DISCUSSION TITLE
🐛 Bug Report: Invalid Origin: Register your new client!

### DISCUSSION THREAD

**Discussion by 0xvashishth** [appwrite/appwrite#4818] [Q&A]
### 👟 Reproduction steps

I have installed AppWrite on GitHub Codespace. And I want to access appwrite console through internet. Appwrite is working perfectly and it's live.

### 👍 Expected behavior

It should create the first account for project console.

### 👎 Actual Behavior

It is not creating my very first user account. It is giving me an error like `Invalid Origin. Register your new client(website link) as a new web platform on your project console dashboard`.

![image](https://user-images.githubusercontent.com/76911582/205684796-bf76c875-3f59-4c42-add2-e1dc4e005663.png)

### 🎲 Appwrite version

Version 1.1.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Github Codespace.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
The `.env` file has 2 environment variables:

1. `_APP_DOMAIN`
2. `_APP_DOMAIN_TARGET`

Would you try changing those to your codespace domain and recreate the containers?

Also, FYI, Appwrite has Gitpod integration so it should spin up quickly and easily with Gitpod.

**Reply by 0xvashishth** [reply]
No Still It is not working :) Same Error...!

```
_APP_DOMAIN=j-imy-cuddly-space-enigma-rvw74qwjpqq3px4x-5000.preview.app.github.dev
_APP_DOMAIN_TARGET=j-imy-cuddly-space-enigma-rvw74qwjpqq3px4x-5000.preview.app.github.dev
```

Here is the whole .env file :)
Website Link(https://j-imy-cuddly-space-enigma-rvw74qwjpqq3px4x-5000.preview.app.github.dev/register)

![image](https://user-images.githubusercontent.com/76911582/205936119-73edf381-c4a1-488e-a787-1a60afe51e4f.png)

Here is the forwarded ports configuration :)
I have tried both port urls but still it is no working..

**Reply by stnguyen90** [reply]
@vasu-1 would you please run `docker compose exec appwrite vars` and verify the environment variables are correct?

**Reply by 0xvashishth** [reply]
I have found the solution :) Thanks for your support 🚀

**Reply by behnamjk** [reply]
would you share your solution ?  I couldn't find how to fix this .

**Reply by Ruboka** [reply]
would appreciate it the answer as for what you did to solve the problem @0xvashishth

**Reply by johnaribatise1217** [reply]
same here @0xvashishth , I am losing my sh*t here

**Reply by 0xvashishth** [reply]
Sorry guys for the late reply, but I tried locally instead of codespace, and it was working properly.

**Comment by PtradeLLC** [comment]
So there is no solution to this issue - Guy said he found the solution, but the  solution he 'found' is to run the app locally? shm
